nav2_bringup/params/nav2_params.yaml

```diff
@@ -291,6 +292,14 @@ planner_server_rclcpp_node:
   ros__parameters:
     use_sim_time: True
 
+smoother_server:
+  ros__parameters:
+    use_sim_time: True
+
+smoother_server_rclcpp_node:
```

Comment:
`smoother_server_rclcpp_node` should not exist anymore, please remove from all yaml files (its in a few)